src/seraphis_main/tx_validation_context.h

```diff
@@ -56,6 +58,12 @@ class TxValidationContext
     TxValidationContext& operator=(TxValidationContext&&) = delete;
 
 //member functions
+    /**
+     * brief: get_sp_ref_set_config - given tx version, return refset config to validate tx against
+     * param: txver -
+     * return: SemanticConfigSpRefSetV1 object which should be used to validate the transaction
+    */
+    virtual SemanticConfigSpRefSetV1 get_sp_ref_set_config(const tx_version_t txver) const = 0;
```

Comment:
```suggestion
    virtual const SemanticConfigSpRefSetV1& get_sp_ref_set_config(const tx_version_t txver) const = 0;
```
Use a ref for > 16 bytes (2 pointers).